src/ServerProviders/Nginx/Plugins.ServerProviders.Nginx/ServerProviderNginx.cs

```diff
@@ -109,31 +111,42 @@ private string GetShellCommandOutput(string command)
 
             process.ErrorDataReceived += (obj, a) =>
             {
-                if (!string.IsNullOrWhiteSpace(a.Data))
+                if (!String.IsNullOrWhiteSpace(a.Data))
```

Comment:
It's not totally consistent but in our codespace I tend to use `string` instead of `String`, probably because some code linting tool said to. Other than being a c# keyword there is no technical difference.

Reply:
I will restore them to `string`